Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:bionic
+FROM ubuntu:20.04
```

Comment:
Can we revert this, so that we're testing against a broader range of environments? Ideally we would still be testing on xenial, but even with a HWE kernel, xenial doesn't have new enough headers in its `linux-libc-dev` to build the `userfaltfd-sys` crate.